tekton/resources/checks/check-pr-has-kind-label.yaml

```diff
@@ -0,0 +1,203 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: github-pull-request-label-checker
+spec:
+  serviceAccountName: gh-pr-label-checker
+  triggers:
+    - name: trigger
+      bindings:
+        - name: trigger-to-check-pr-labels
+      template:
+        name: check-pull-request-labels
+---
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: check-pull-request-labels
+spec:
+  params:
+  - name: labels
+    description: List of labels currently on the Pull Request
+    type: string
+  - name: gitRepository
+    type: string
+  - name: pullRequestUrl
+    type: string
+  resourcetemplates:
+  - apiVersion: tekton.dev/v1alpha1
+    kind: PipelineResource
+    metadata:
+      name: pr-$(uid)
+    spec:
+      type: pullRequest
+      params:
+        - name: url
+          value: $(params.pullRequestUrl)
+      secrets:
+        - fieldName: authToken
+          secretName: github-token
+          secretKey: oauth
+  - apiVersion: tekton.dev/v1alpha1
+    kind: PipelineResource
+    metadata:
+      name: git-source-$(uid)
+    spec:
+      type: git
+      params:
+      - name: revision
+        value: master
+      - name: url
+        value: $(params.gitRepository)
+      - name: labels
+        value: $(params.labels)
+  - apiVersion: tekton.dev/v1alpha1
+    kind: TaskRun
+    metadata:
+      name: check-pr-labels-$(uid)
+    spec:
+      taskSpec:
+        inputs:
+          resources:
+          - name: source
+            type: git
+          - name: pr
+            type: pullRequest
+          params:
+          - name: labels 
+            description: The labels attached to the Pull Request
+          - name: gitRepository
+            description: The URL of the git repository
+          - name: pullRequestUrl
+            description: The URL of the Pull Request
+        outputs:
+          resources:
+          - name: pr
+            type: pullRequest
+        steps:
+        - name: copy-pr-to-output
+          image: busybox
+          script: |
+            #!/bin/sh
+            mkdir -p $(outputs.resources.pr.path)
+            cp -r $(inputs.resources.pr.path)/* $(outputs.resources.pr.path)/
+        - name: check-labels
+          env:
+          - name: GH_LABELS
+            value: '$(params.labels)'
+          image: python:3-alpine
+          script: |
+            #!/usr/bin/env python
+            import json
+
+            labels = json.loads('$(params.labels)')
+            labelNames = list(map(lambda e: e["name"], labels))
+            kindLabels = list(filter(lambda e: "kind" in e, labelNames))
+            comment_template=""
+            if (len(kindLabels) > 1 or len(kindLabels) == 0):
+              print('failed')
+              comment_template += """
+            **This PR cannot be merged:** expecting one kind/ label
+
+            <details>
+            The following `kind/` labels are available:</p>
+
+            These are some kind labels
```

Comment:
This list should ideally be pulled from the configuration in the plumbing repo, so that it's always up to date.

Reply:
This has been fixed